libsbefifo/operation.c

```diff
@@ -152,10 +156,10 @@ int sbefifo_operation(struct sbefifo_context *sctx,
 		return ENOTCONN;
 
 	/*
-	 * Allocate extra memory for FFDC (SBEFIFO_MAX_FFDC_SIZE = 0x2000)
+	 * Allocate extra memory for FFDC (SBEFIFO_MAX_FFDC_SIZE = 0x8000)32kb
 	 * Use *out_len as a hint to expected reply length
 	 */
-	buflen = (*out_len + 0x2000 + 3) & ~(uint32_t)3;
+	buflen = (*out_len + 0x8000 + 3) & ~(uint32_t)3;
```

Comment:
Is there a challenge or overhead in creating a const variable for 0x8000 like SBEFIFO_MAX_FFDC_SIZE?  though we have added a comment with the same value?